src/audio-recorder.js

```diff
@@ -78,75 +78,79 @@ export class AudioRecorder extends HTMLElement {
           padding: 5px 10px 5px 5px;
           max-width: 800px;
         }
-        
+
+        #microphone-input {
+          max-width: 100px;
+        }
```

Comment:
Suggestion for default styles:

`background-color: buttonface;`
`border: none;`